widgetssdk/src/main/java/com/glia/widgets/chat/adapter/ChatAdapter.kt

```diff
@@ -259,7 +333,15 @@ internal class ChatAdapter(
         const val VISITOR_IMAGE_VIEW_TYPE = 7
         const val NEW_MESSAGES_DIVIDER_TYPE = 8
         const val SYSTEM_MESSAGE_TYPE = 9
-        const val CUSTOM_CARD_TYPE = 10 // Should be the last type with the highest value
+
+        //GVA Types
+        const val GVA_RESPONSE_TEXT_TYPE = 10
+        const val GVA_PERSISTENT_BUTTONS_TYPE = 11
+        const val GVA_QUICK_REPLIES_TYPE = 12
+        const val GVA_GALLERY_CARDS_TYPE = 13
+
+        //Custom Card
+        const val CUSTOM_CARD_TYPE = 1000 // Should be the last type with the highest value
```

Comment:
Is this works well for custom cards?

Reply:
Probably it should.

Reply:
I thought of somehow keeping us from potential bugs, in case we forget to increment `CUSTOM_CARD_TYPE` with the new type of addition. But I think that adding new types is happening not often enough, so we can keep it as it was designed.

Reply:
I checked. 1000 isn't working for the custom cards.

Reply:
fixed